src/main/java/core/basesyntax/serviceimpl/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.serviceimpl;
+
+import core.basesyntax.db.FruitStorage;
+import core.basesyntax.service.ReportGenerator;
+import java.util.Map;
+
+public class ReportGeneratorImpl implements ReportGenerator {
+    private static final String REGEX_TO_SPLIT = ",";
+    private static final String CONTENTS = "fruit, quantity";
+
+    @Override
+    public String createReport() {
+        StringBuilder stringBuilder = new StringBuilder(CONTENTS + System.lineSeparator());
+        for (Map.Entry<String, Integer> entry : FruitStorage.fruitRepository.entrySet()) {

```

Comment:
Direct static access to `fruitRepository` from outside its own class `FruitStorage` breaks the encapsulation principle. Consider adding a method in `FruitStorage` to get the needed data.